### PR TITLE
fc, pwd: add examples

### PR DESCRIPTION
**Pull request by IranNeto**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/fc.md [hunk 1]
```diff
@@ -15,3 +15,11 @@
...
 
 `fc -l`
+
+- List recent commands in reverse order:
+
+`fc -r`
+
+- List commands in a given interval:
+
+`fc {{'416'}} {{'420'}}`
```
Comment (navarroaxel) [inline pages/common/fc.md:25]: I don' know if the `'`s are required, but should be outside the curly braces
```suggestion
`fc '{{416}}' '{{420}}'`
```

**Review by navarroaxel — APPROVED** [review]

**Review by mfrw — APPROVED** [review]

**Comment by lengyijun** [comment]
```
  - List recent commands in reverse order:
    fc -r
```

 `fc -r` will edit the last command, according to man page

**Comment by mfrw** [comment]
Thank you for the catch.
Would you mind sending a PR, otherwise i can send one once I get sometime

**Comment by lengyijun** [comment]
#9805

**Pull request merged**